wrappers/android/librealsense/src/main/java/com/intel/realsense/librealsense/Sensor.java

```diff
@@ -21,8 +21,14 @@
     public <T extends Sensor> T as(Extension extension) {
         switch (extension){
```

Comment:
Assumes that the sensor is extendable -- need to catch and throw an error if not extendable...

Reply:
corrected